app/domain/errors.rb

```diff
@@ -353,9 +353,11 @@ module AuthnGcp
         code: "CONJ00067E"
       )
 
-      TokenClaimNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
-        msg:  "Claim '{0-claim-name}' not found or empty in token. " \
-              "Verify that you requested the token using 'format=full'",
+      ResourceRestrictionNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
```

Comment:
@shulifink @InbalZilberman  please review, an example to the message from the log:
`Authentication Error: #<Errors::Authentication::AuthnGcp::ResourceRestrictionNotFoundOrEmpty: CONJ00068E Resource restriction 'instance-name' does not exists resource in JWT token. Verify that you configured the host with permitted restrictions. In case of Compute Engine token verify that you requested the token using 'format=full' >`

Reply:
Resource restriction 'instance-name' does not exists in JWT token under XpathX. Verify that you configured the host with permitted restrictions. In case of Compute Engine token verify that you requested the token using 'format=full'

Reply:
@InbalZilberman we dont have the path in this stage since we already decoded the token, and it is very easy to understand it from previous log messages, Do you still think it is important to change the code for it?

Reply:
What do you mean by "does not exists resource in JWT token."?

Reply:
It is an actually was an approved message i just added the 2 additional lines in the end,
For your question it means that: the JWT token doesn't contain a corresponding resource restriction which configured on the conjur host.

For example the host configured with the following email as resource restriction:
```
- !host
  id: hosttest1
  annotations:
    authn-gcp/service-account-email: refreshing-mark-284016@appspot.gserviceaccount.com
```
 
We are expecting that the JWT token will contain an email claim with the above value,
But the email claim not exists in the JWT token so the host is unauthorized